### PR TITLE
Docs: Adds pgnet and database webhooks

### LINKED ISSUE #73: How does pg_net fit into the overall supabase architecture?
Hey,

we are making quite heavy use of pg_net, mainly to make a request to an external api from within a trigger. For some of these, we will now migrate to a postgres-based queue (postgraphile-worker / pg-boss) to have better observability.

During the migration, I began to wonder how pg_net fits into the supabase architecture, for what use-cases it is supposed to be used and what its limitations are, especially regarding performance.

As an example, we are using segment to collect usage data and want to track events from within the database. One option would be to use pg_net to make a request to the Segment API. We created a simple segement.track function which makes the request using pg_net. When our usage grows, we might get a few events per seconds. How much requests is pg_net supposed to handle? As an alternative, we could either send the requests batch-wise or use a queueing solution and drop pg_net. Further, a webhook or queuing plugin for realtime would probably be a more scalable alternative. 

Is there any official guidance on the scalability and intend of pg_net?

Best
Philipp

### PR DESCRIPTION
**Pull request by kiwicopple**
fix: https://github.com/supabase/pg_net/issues/73

### CODE REVIEW

File: apps/reference/docs/guides/database/database-webhooks.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+id: database-webhooks
+title: 'Database Webhooks'
+description: Trigger external payloads on database events.
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+Database Webhooks allow you to send real-time data from your database to another whenever a table event occurs.
```
Comment (steve-chavez) [inline apps/reference/docs/guides/database/database-webhooks.mdx:10]: Not necessarily to another database right?
```suggestion
Database Webhooks allow you to send real-time data from your database to another system whenever a table event occurs.
```

File: apps/reference/docs/guides/database/extensions/pgnet.mdx [hunk 2]
```diff
@@ -0,0 +1,207 @@
+---
+id: pgnet
+title: 'pg_net: Async Networking'
+description: 'pg_net: an async networking extension for PostgreSQL.'
+---
+
```
Comment (steve-chavez) [inline apps/reference/docs/guides/database/extensions/pgnet.mdx:6]: Could we add a notice here that pg_net is still not stable yet? The interface will likely change to the one proposed on https://github.com/supabase/pg_net/issues/62 and functions will not require security definer.

```suggestion
:::caution

pg_net is not at 1.0(stable) yet. Expect the function signatures to change.
```

File: apps/reference/docs/guides/database/extensions/pgnet.mdx [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+:::caution
+
+The pg_net API is not at 1.0 (stable). Functions signatures may change.
+
+:::
+
```
Comment (kiwicopple) [inline apps/reference/docs/guides/database/extensions/pgnet.mdx:15]:
```suggestion
```

File: apps/reference/docs/guides/database/database-webhooks.mdx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+Database Webhooks allow you to send real-time data from your database to another system whenever a table event occurs.
+
+:::note
+Database Webhooks were previously known as Function Hooks.
+:::
```
Comment (kiwicopple) [inline apps/reference/docs/guides/database/database-webhooks.mdx:14]:
```suggestion
```

File: apps/reference/docs/guides/database/extensions/pgnet.mdx [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+title: 'pg_net: Async Networking'
+description: 'pg_net: an async networking extension for PostgreSQL.'
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+:::caution
+
+The pg_net API is not at 1.0 (stable). Functions signatures may change.
```
Comment (dannykng) [inline apps/reference/docs/guides/database/extensions/pgnet.mdx:12]: Maybe this should say it's still alpha/beta instead so it sounds a bit less negative.

**Review by dannykng — APPROVED** [review]
Just one more suggestion but LGTM.

File: apps/reference/docs/guides/database/extensions/pgnet.mdx (hunk 5 above)
Comment (kiwicopple) [inline apps/reference/docs/guides/database/extensions/pgnet.mdx:12]:
```suggestion
The pg_net API is in beta. Functions signatures may change.
```

**Pull request merged**